examples/drivenet/drivenet_train.ipynb

```diff
@@ -0,0 +1,325 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to DriveNet\n",
+    "\n",
+    "DriveNet is a CNN able to drive an agent on a specific route. Among its features, DriveNet can follow the midlane and respect traffic lights.\n",
+    "\n",
+    "### Imitation Learning and Its limitations\n",
+    "DriveNet is trained using imitation learning on our Lyft L5 Prediction Dataset 2020. We feed examples of real driving experience to the model and we expect it to take the same actions as the driver did in those episodes. This is very similar to how tasks like classification are usually solved.\n",
+    "\n",
+    "Imitiation Learning is powerful, but it has a strong limitation. Even with a lot of data, is easy to drift away from the original train distribution. This may not be an issue for single-shot task (like classification), but this limit becomes immediately clear for autoregressive predictions.\n",
+    "\n",
+    "After training DriveNet, we would like it to take full control of the AV and determine its future locations. This means that subsequent predictions will be use as future locations for the AV. \n",
+    "\n",
+    "In this scenario it's very easy to drift away from the original distribution in the train set. During training our DriveNet has seen good examples of driving only. In particular, this means **almost perfect midlane following**. However, even a small constant vertical displacement during the close loop test can accumulate enough error to lead the AV completely out of its distribution.\n",
+    "\n",
+    "In this notebook you will see an example of how we can tackle this issue effectively and learn to recover from out-of-distribution situations.\n",
+    "\n",
+    "TODO PICTURE OF WHAT DRIFTING MEANS HERE\n",
+    "\n",
+    "\n",
+    "In this notebook you're going to train your own Drivenet model using Lyft L5 Dataset and L5Kit.\n",
```

Comment:
the ... Dataset - it's called differently though, isn't it?

Reply:
I can remove prediction, but that's the name of the dataset in the end xD